File: src/problem6/SumSquareDifference.java
package problem6;

public class SumSquareDifference {
	
	private static final int LIMIT = 100;
	
	private static int getSumSquares() {
		int sum = 0;
		for (int i = 1; i <= LIMIT; i++) {
			int square = i*i;
			sum += square;
		}
		
		return sum;
	}
	
	private static int getSquareSums() {
		int sum = 0;
		for (int i = 1; i <= LIMIT; i++) {
			sum += i;
		}
		int square = sum*sum;
		
		return square;
	}

	public static void main(String[] args) {
		int sumSquares = getSumSquares();
		int squareSums = getSquareSums();
		int diff = squareSums - sumSquares;
		
		System.out.println(diff);
	}

}

File: src/Python/Page 1/Problem 6/sum_square_difference.py
def main():
    limit = 100
    sum_squares = 0
    square_sum = 0
    for i in range(1, limit+1):
        sum_squares += i ** 2
        square_sum += i
    square_sum **= 2
    diff = square_sum - sum_squares
    print('The difference between the sum of the squares and the square of the sum is: {}'.format(diff))


if __name__ == '__main__':
    main()
